.github/workflows/run_tests.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        go_versions: [ '1.20', '1.21', '1.22' ]
+        go_versions: [ '1.21', '1.22', '1.23' ]
```

Comment:
1.23 is a separate change, but we need to adapt to our new policy and drop one version from the set.

Reply:
Removed 1.23 and reduced to just couple versions